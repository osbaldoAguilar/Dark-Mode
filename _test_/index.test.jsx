// __tests__/index.test.jsx

import { render, screen } from '@testing-library/react'
import Home from '../pages/index'

describe('Home', () => {
    it('renders a unchanged', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
    })

    it('renders a component', () => {
      const { main } = render(<Home/>)
      expect(main).toMatchSnapshot()
    })
})
