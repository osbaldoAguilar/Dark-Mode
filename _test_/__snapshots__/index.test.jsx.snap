// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a component 1`] = `undefined`;

exports[`Home renders a unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <h2
        class="title"
      >
        <a
          href="https://github.com/osbaldoAguilar/Dark-Mode"
        >
          Osbaldo Aguilar: Resume Creator!
        </a>
      </h2>
      <div>
        <h4>
          Reason and Problem to resolve: 
        </h4>
        <p>
          Hello I was having issues coming up with ideas to show off the skills I have acquired over the last 3 years. I was having trouble coming up with mini projects with defined results. Here's my first attempt. Fill out a form to create a Resume that you can download as a PDF!!
        </p>
      </div>
      <div>
        <div
          class="btns"
        >
          Click Me To See Created Resume
        </div>
      </div>
    </main>
    <footer
      class="footer"
    />
  </div>
</div>
`;
